hoodie-common/src/main/java/com/uber/hoodie/common/util/collection/converter/DefaultKeyConverter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2016 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.uber.hoodie.common.util.collection.converter;
+
+import com.twitter.common.objectsize.ObjectSizeCalculator;
+
+/**
+ * A default converter implementation for String type of payload key
+ */
+public class DefaultKeyConverter implements Converter<String> {
```

Comment:
can we rename to `StringConverter`?  this way we can use the class, as long its a string..

Reply:
good point, done.